examples/Batch/WordCountFSharp/WordCountExample.fs

```diff
@@ -0,0 +1,34 @@
+﻿open Microsoft.Spark.CSharp.Core
+open Microsoft.Spark.CSharp.Services
+open System.Reflection
+open System.Collections.Generic
+
+[<EntryPoint>]
+let main = function
+    | [| filePath |] ->
+        let logger =
+            LoggerServiceFactory.SetLoggerService Log4NetLoggerService.Instance
+            LoggerServiceFactory.GetLogger (MethodInfo.GetCurrentMethod().DeclaringType)
+        
+        let sparkContext = SparkConf().SetAppName "MobiusWordCount" |> SparkContext
+        logger.LogInfo (sprintf "Reading from file %s" filePath)
+
+        try
```

Comment:
It feels a shame you need this ceremony of a "main" and a "try"/"catch".  It feels like this whole thing should just be a script (or a .fs in the style of a script).  Speaking of which does this work with F# interactive? thx